glean-core/android/src/test/java/mozilla/telemetry/glean/acmigration/GleanDataMigrationTest.kt

```diff
@@ -55,29 +56,48 @@ class GleanDataMigrationTest {
         prefs?.edit()?.putInt("${pingName}_seq", number)?.apply()
     }
 
+    /**
+     * A convenience function to make `setInitialDataToMigrate` more readable.
+     */
+    private fun setSharedPrefsData(
+        context: Context,
+        storageName: String,
+        putFunc: (SharedPreferences.Editor) -> SharedPreferences.Editor
+    ) {
+        context.getSharedPreferences(
+                "${GleanACDataMigrator.GLEAN_AC_PACKAGE_NAME}.storages.$storageName",
+                Context.MODE_PRIVATE)
+            .edit()
+            .let { putFunc(it) }
+            .apply()
+    }
+
     private fun setInitialDataToMigrate(context: Context, firstRunDate: String) {
         // Set a fake sequence number for the baseline ping.
         setFakeSequenceNumber(context, "baseline", TEST_BASELINE_SEQ)
 
         // Set a previously existing client id.
-        context
-            .getSharedPreferences(
-                "${GleanACDataMigrator.GLEAN_AC_PACKAGE_NAME}.storages.UuidsStorageEngine",
-                Context.MODE_PRIVATE
-            )
-            .edit()
-            .putString("glean_client_info#client_id", TEST_CLIENT_ID)
-            .apply()
+        setSharedPrefsData(context, "UuidsStorageEngine") {
+            it.putString("glean_client_info#client_id", TEST_CLIENT_ID)
+        }
 
         // Set a previously existing first_run_date.
-        context
-            .getSharedPreferences(
-                "${GleanACDataMigrator.GLEAN_AC_PACKAGE_NAME}.storages.DatetimesStorageEngine",
-                Context.MODE_PRIVATE
-            )
-            .edit()
-            .putString("glean_client_info#first_run_date", firstRunDate)
-            .apply()
+        setSharedPrefsData(context, "DatetimesStorageEngine") {
+            it.putString("glean_client_info#first_run_date", firstRunDate)
+        }
+
+        // Set a test labeled_counter in the baseline ping, for convenience.
```

Comment:
You can leave this part out, it was a WIP portion of my PR :)

Reply:
Gotcha!